Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -98,25 +98,62 @@ static bool FindPromotionMove(Board board, IReadOnlyList<Move> moves, out Move p
     #endregion
 
     #region Capture
-    static bool FindCaptureMove(Board board, IReadOnlyList<Move> moves, out Move captureMove)
+    bool FindCaptureMove(Board board, IReadOnlyList<Move> moves, out Move captureMove)
+    {
+        captureMove = TryGetCaptureMove(board, moves, AfraidOfLosing);
+        return captureMove.IsCapture;
+    }
+
+    static Move TryGetCaptureMove(Board board, IReadOnlyList<Move> moves, bool lookForTwoMovesCapture = true, bool allowSacrifice = true)
```

Comment:
oh, `lookForTwoMovesCapture` makes the job done, but probably it is worth to start thinking on `TODO 3` since current approach is not scalable.